ports/esp32/network_wlan.c

```diff
@@ -711,6 +711,9 @@ STATIC const mp_rom_map_elem_t wlan_if_locals_dict_table[] = {
     { MP_ROM_QSTR(MP_QSTR_PM_NONE), MP_ROM_INT(WIFI_PS_NONE) },
     { MP_ROM_QSTR(MP_QSTR_PM_PERFORMANCE), MP_ROM_INT(WIFI_PS_MIN_MODEM) },
     { MP_ROM_QSTR(MP_QSTR_PM_POWERSAVE), MP_ROM_INT(WIFI_PS_MAX_MODEM) },
+    #ifdef MICROPY_PY_NETWORK_MODULE_GLOBALS_INCLUDEFILE
+    #include MICROPY_PY_NETWORK_MODULE_GLOBALS_INCLUDEFILE
+    #endif
```

Comment:
This will include everything, including LAN constants, which is wrong.

I suggest to just copy across the relevant WLAN constants.

Reply:
Okay